companion/lib/Surface/Types.ts

```diff
@@ -73,6 +73,7 @@ export interface SurfacePanelEvents {
 
 	click: [x: number, y: number, pressed: boolean, pageOffset?: number]
 	rotate: [x: number, y: number, rightward: boolean, pageOffset?: number]
+	swipePage: [forward: boolean]
```

Comment:
as mentioned elsewhere, I think this should renamed to `changePage`. that will allow other surfaces to use it in a way that makes sense for them without the naming becoming confusing